smileidentity/web-client
# redesign(eSignature): modify file links styles to show interactivity

# Steps to Test
- Test on https://smileid-e-signature.glitch.me
- Verify that you see the download icon and that the document link changes background-color on hover
